### DISCUSSION TITLE
Pinecil heats up when starting, but temp remains stable

### DISCUSSION THREAD

**Discussion by skorokithakis** [Ralim/IronOS#1832] [Q&A]
Hello,
my Pinecil v1 heats up to a red hot tip, but the temperature keeps showing 25-26 C. Someone told me that that means that a part is defective, and I have a spare PCB with a busted LDO. I'm wondering if someone can tell me which part is defective on the thermal runaway Pinecil, so I can replace it with one from the broken PCB.

I would be very grateful if someone could post the part markings or, even better, a photo.

**Comment by Ralim** [comment]
The failure is almost always the power control mosfet. Its an 8 pin SOIC IC. Search the wifi for "MOSFET" for the part numbers for the model, its also a really cheap part to buy a replacement for generally -> https://wiki.pine64.org/wiki/Pinecil:_Test,_Repair,_Issues#Schematics_and_Board_Data

**Reply by Ralim** [reply]
The community chat is great for helping with this sort of thing

**Reply by skorokithakis** [reply]
Thank you! I searched for "community" and "discord" in the README and docs but didn't find anything, can you point me to it?

**Reply by Ralim** [reply]
Its hosted by Pine64 Community, not IronOS so refer to their website https://www.pine64.org/

**Reply by Ralim** [reply]
IronOS only has github as the official support medium, but the Pine64 chat has a bunch of really helpful people in it, and its generally faster than me responding to tickets.

**Reply by skorokithakis** [reply]
Makes sense, thanks!

**Comment by skorokithakis** [comment]
Thanks! I see two SOICs on the board, U8 and U13. I assume the U8 is the tip one? The other PCB has probably blown U13, so I should just stock up on these and have two working irons, thanks!

**Reply by skorokithakis** [reply]
Looks like U8 is a step-down converter, the broken Pinecil had 25V applied so that might have fried the MOSFET. Looks like I won't be able to use that as a spare, sadly, and I can't find any MOSFET parts on AliExpress with that model number. Maybe someone on chat can help, I'll try to find how to join.

**Reply by Ralim** [reply]
Ah, yeah its not the step-down. Sorry, forgot that one was also a SOIC-8.

Its a very common mosfet pinout, so there are lots of interchangable ones on mouser. Its somewhere on here in an issue but also people often swap in the higher rated one from the PinecilV2

**Reply by skorokithakis** [reply]
I'll look, thank you!
